frontend/src/Explorer/View/Dashboard/Blocks.purs

```diff
@@ -107,12 +112,38 @@ blockRow state (CBlockEntry entry) =
     P.link (toUrl <<< Block $ entry ^. cbeBlkHash)
         [ P.className "blocks-body__row" ]
         [ blockColumn <<< show $ entry ^. cbeHeight
-        , blockColumn <<< show <<< unwrap $ entry ^. (cbeTimeIssued <<< _NominalDiffTime)
+        , blockColumn (prettyDuration language (Milliseconds $ unwrap $ entry ^. (cbeTimeIssued <<< _NominalDiffTime)))
+        -- , blockColumn <<< show <<<  $ unwrap $ entry ^. (cbeTimeIssued <<< _NominalDiffTime)
         , blockColumn <<< show $ entry ^. cbeTxNum
         , blockColumn <<< show $ entry ^. (cbeTotalSent <<< _Coin <<< getCoin)
-        , blockColumn <<< fromMaybe (translate (I18nL.common <<< I18nL.cUnknown) $ state ^. lang) $ entry ^. cbeRelayedBy
+        , blockColumn <<< fromMaybe (translate (I18nL.common <<< I18nL.cUnknown) language) $ entry ^. cbeRelayedBy
         , blockColumn <<< show $ entry ^. cbeSize
         ]
+    where
+      language      = state ^. lang
+
+-- cDays cHours cMinutes cSeconds
+prettyDuration :: forall a. Duration a => Language -> a -> String
```

Comment:
Could you move `prettyDuration` into `Explorer.Util.String` or `Explorer.Util.Time` or similar?